monitoring/serializers.py

```diff
@@ -40,11 +44,15 @@ class Meta:
         id_field = False
         fields = (
             'id',
-            'tb_ciclo_monitoramento_id',
             'no_imagem',
             'dt_imagem',
             'no_estagio',
-            'dt_cadastro',
+            'no_ciclo',
```

Comment:
Here is fine, because of the detailed request.

Reply:
OK